components/support/nimbus-fml/src/editing/error_path.rs

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#[derive(Clone)]
+pub(crate) struct ErrorPath {
+    pub(crate) literals: Vec<String>,
```

Comment:
Can you document these members? I am unclear about what `literals` is, especially in the `array_index` method.